CHANGELOG.md

```diff
@@ -1,5 +1,40 @@
 # Changelog
 
+## 0.1.33
+
+### Breaking changes
+
+Using the keyring requires a username to be provided on index URLs now. Previously, the username `oauth2accesstoken`
+was assumed. This will affect Google Artifact Registry users using `--keyring-provider subprocess` and an index URL
+without a username. The suggested fix is to add the required username to index URLs, e.g., `https://oauth2accesstoken@<url>`.
+
+See [#2976](https://github.com/astral-sh/uv/pull/2976#discussion_r1566521453) for details.
+
+### Enhancements
+
+- Allow passing a virtual environment path to `uv pip --python` ([#3064](https://github.com/astral-sh/uv/pull/3064))
+- Add compatibility argument for `pip list --outdated` ([#3055](https://github.com/astral-sh/uv/pull/3055))
+- Add support for `.tar.bz2` source distributions ([#3069](https://github.com/astral-sh/uv/pull/3069))
```

Comment:
Note that this is a fallback, we don't support-support (https://packaging.python.org/en/latest/specifications/source-distribution-format/#source-distribution-file-name) these but just don't error anymore when they exist in a legacy registry. Uploading to pypi is banned (https://github.com/pypi/warehouse/blob/ae35f55aa9ae6250e7a698e34dbe64fbbf54632b/tests/unit/forklift/test_legacy.py#L1556)

Reply:
Thanks!